crates/web5/src/apid/credentials/verifiable_credential_11.rs

```diff
@@ -0,0 +1,123 @@
+use super::Result;
+use crate::apid::dsa::{Signer, Verifier};
+use core::fmt;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::HashMap,
+    fmt::{Display, Formatter},
+    sync::Arc,
+};
+
+const BASE_CONTEXT: &str = "https://www.w3.org/2018/credentials/v1";
+const BASE_TYPE: &str = "VerifiableCredential";
+
+#[derive(Serialize, Deserialize, Debug, Default, Clone, PartialEq)]
+pub struct NamedIssuer {
+    pub id: String,
+    pub name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(untagged)]
+pub enum Issuer {
+    String(String),
+    Object(NamedIssuer),
+}
+
+impl<I> From<I> for Issuer
+where
+    I: Into<String>,
+{
+    fn from(s: I) -> Self {
+        Issuer::String(s.into())
+    }
+}
+
+impl Display for Issuer {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        match self {
+            Issuer::String(s) => write!(f, "{}", s),
+            Issuer::Object(ni) => write!(f, "{}", ni.id),
+        }
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct VerifiableCredential {
+    #[serde(rename = "@context")]
+    pub context: Vec<String>,
+    pub id: String,
+    #[serde(rename = "type")]
+    pub r#type: Vec<String>,
+
+    // 🚧 UDL support
+    // pub issuer: Issuer,
```

Comment:
if it stays a problem, we can make issuer a string only for now.
maybe need to get a issue for this.

Otherwise I will update it along with the josekit changes

Reply:
Oh perfect I see Issuer is string as of now, lol nice